mimic/test/test_nova.py

```diff
@@ -1261,15 +1284,17 @@ def test_create_server_failure_based_on_metadata(self):
             treq.json_content(failing_create_response)
         )
 
-        self.assertEquals(failing_create_response_body['message'],
-                          "Sample failure message")
-        self.assertEquals(failing_create_response_body['code'], 503)
+        self.assertEquals(
+            failing_create_response_body['specialType']['message'],
+            "Sample failure message")
+        self.assertEquals(
+            failing_create_response_body['specialType']['code'], 503)
 
-    def test_create_false_negative_failure_using_behaviors(self):
+    def _try_false_negative_failure(self, failure_type=None):
         """
-        :func:`create_server` fails with given error message and response
-        code, but creates the server anyway, when a behavior is registered
-        that matches its hostname.
+        Helper function to list servers and verify that there are no servers,
```

Comment:
Once again, starting things with "helper function" is an anti-pattern (albeit one I do myself sometimes too) and this should just say what it does without qualifying its role as a function or helper or utility or whatever.